<launch>
<node name="dynamixel_sdk_bulk_rw" pkg="dynamixel_sdk" type="bulk_rw" respawn="true" output="screen" />
</launch>
